styles/hero.css

```diff
@@ -3,10 +3,18 @@
     padding: 20px 40px;
 }
 
+.hero-container {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    gap: 1rem;
+}
+
 .hero-image {
-    border-radius: 170.859px;
     text-align: center;
-    margin-bottom: 1rem;
+}
+
+.image {
+    /* width: 50%; */
```

Comment:
- [ ] If you have defined CSS classes in your stylesheet but haven't applied any styles to them, it's a good practice to remove those class selectors entirely. This helps in reducing the file size and keeps your CSS codebase tidy.